src/main/flight/pid.c

```diff
@@ -612,8 +627,28 @@ void FAST_CODE pidController(const pidProfile_t *pidProfile, const rollAndPitchT
         }
 
         // -----calculate I component
+        float itermErrorRate;
+        if (itermRelax && (axis < FD_YAW || itermRelax == 2 )) {
```

Comment:
This should use an enum to make it obvious what the `itermRelax` values stand for.